generate_documentation_qmd/action.yml

```diff
@@ -0,0 +1,95 @@
+name: 'Build and publish qmd'
+author: 'Dries Schaumont'
+description: >
+  Create quarto markdown files for viash components or workflows, based on a jinja template.
+inputs:
+  input_dir:
+    required: false
+    description: 'Directory where the viash configs can be found.'
+    default: "."
+  component_template:
+    required: false
+    description: >
+      Jinja template that can be used to customize the resulting qmd files.
+      By default will use the included template.
+  output_dir:
+    required: true
+    description: >
+      Output folder to write to.
+  dest_path:
```

Comment:
`dest_path` is not used any of the steps, or am I mistaken?

Reply:
Good catch, added!